modules/mcuboot/CMakeLists.txt

```
@@ -264,6 +258,32 @@ if(CONFIG_BOOTLOADER_MCUBOOT)
         net_core_app_sign_target
         )
 
+      generate_dfu_zip(
+        TARGET mcuboot_sign_target
+        OUTPUT ${PROJECT_BINARY_DIR}/dfu_application.zip
+        BIN_FILES ${PROJECT_BINARY_DIR}/${app_core_binary_name} ${PROJECT_BINARY_DIR}/${net_core_binary_name}
+        TYPE application
+        SCRIPT_PARAMS
+        "${app_core_binary_name}load_address=$<TARGET_PROPERTY:partition_manager,PM_APP_ADDRESS>"
+        "${net_core_binary_name}load_address=missing"
+        "${app_core_binary_name}version_MCUBOOT=${CONFIG_MCUBOOT_IMAGE_VERSION}"
+        "${net_core_binary_name}version_BN0=${hci_rpmsg_CONFIG_FW_INFO_FIRMWARE_VERSION}"
```

This doesn't work so we will have to share this through shared vars.
I'm not really sure what the best place to set these are but currently I set 
```
share("set(${IMAGE_NAME}soc ${CONFIG_SOC}")
share("set(${IMAGE_NAME}fw_version ${CONFIG_FW_INFO_FIRMWARE_VERSION}")
```

This will allow you to do
```suggestion
         "${net_core_binary_name}version_BN0=${hci_rpmsg_fw_version}"
 	 "${net_core_binary_name}soc=${hci_rpsmg_soc}"
```